apps/src/scratch/scratch.js

```diff
@@ -14,7 +14,7 @@ import { singleton as studioApp } from '@cdo/apps/StudioApp';
 import ScratchView from './ScratchView';
 import { scratchDefaultProject } from './scratchDefaultProject';
 
-const Scratch = window.Scratch = window.Scratch || {};
+const Scratch = window.__TestInterface__ = {};
```

Comment:
ugh. Looks like sometimes we use __TestInterface__ and sometimes __TestInterface. The latter seems like the more common one.